### PR TITLE
Zendesk WIP: New Help Page

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR updates the design of the Help page. Since it was basically a small re-write, I took the opportunity to convert the activity to Kotlin. Aside from porting the existing functionality to the new design, the only addition is "Contact email" which is implemented in #7825. That PR needs to be merged before this PR can be reviewed. (and the base branch needs to be updated)

To Test:

* Make sure the existing functionality works exactly the same way it used to (basically opening new pages 🤷‍♂️ )
* If you do a fresh install, you should see a "Not Set" for the "Contact email" but when you tap on it, the dialog will be pre-populated the same way it'll be when you tap on the "Contact us" button.
* Make sure saving the dialog updates the "Contact email" field

Here are some screenshots:

screenshot_1527624135
screenshot_1527624918
screenshot_1527624123

@SylvesterWilmott The Zeplin project had the page title `Support` whereas we used to have `Help and Support`. The title for the button of support page is `Help & Support`. Could we pick one of them and just use it in both the button title and the page title? I prefer the ones that include `Help` fwiw.

### CODE REVIEW

**Comment by oguzkocer** [comment]
@SylvesterWilmott @aerych @ScoutHarris One thing to keep in mind with this current design is that, if the contact email is not set and the user changes it directly from the Help page (instead of going through Contact Us, My Tickets etc) the support name will be empty and they won't have a chance to change it.

**Comment by SylvesterWilmott** [comment]
> The title for the button of support page is Help & Support. Could we pick one of them and just use it in both the button title and the page title? I prefer the ones that include Help fwiw.

What you have in the screenshot looks good!

> One thing to keep in mind with this current design is that, if the contact email is not set and the user changes it directly from the Help page (instead of going through Contact Us, My Tickets etc) the support name will be empty and they won't have a chance to change it.

Yeah you are right, I've overlooked this case. Can we possibly add a new field giving the user the ability to edit their contact name from here as well?

</img>

**Comment by oguzkocer** [comment]
> Can we possibly add a new field giving the user the ability to edit their contact name from here as well?

I am not sure that's the best way to go. I'd rather not reset the tickets on the device for a name change that doesn't have a significant benefit. But that might be just me.

One idea I had was to combine the contact information into a single field and always show the same dialog with both email and name input. So, instead of the "Contact email" title in the original design, we'll have "Contact information" and as the text underneath, instead of just the email, we'll have both the email and name. Something like `John Doe (john@doe.example)` or some other format.

This doesn't completely fix the issue with resetting tickets, but it _might_ help since I'd expect users to change their email and name at the same time. It'll also simplify the implementation code.

We can also leave it as is and not let them change the name at all.

@SylvesterWilmott It's your call though, I just wanted to lay out my thoughts.

-------------------------------------

I just realized that we still have the `To continue please enter your email address` message for the `Contact email` in the Help page. I feel like that's probably unnecessary. Should we just show a title like `Contact email` when it's being changed? Or a different kind of explanation maybe?

Saying `To continue` feels weird to me as there is nothing to continue to in this case.

**Comment by SylvesterWilmott** [comment]
I don't think it's important enough to allow everyone to change their name at any time (and risk losing tickets) to accommodate the minority who will enter their email directly. Lets take the simple route and don't allow name changes. Thoughts @aerych ?

**Comment by oguzkocer** [comment]
@0nko This PR is now ready to review :) Thanks again for taking these 🙇

**Comment by aerych** [comment]
> Lets take the simple route and don't allow name changes. Thoughts @aerych ?

Agreed. I'm reluctant to do anything that increases the risk of tickets becoming unavailable in the app.

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+        val actionBar = supportActionBar
+        if (actionBar != null) {
+            actionBar.setHomeButtonEnabled(true)
+            actionBar.setDisplayHomeAsUpEnabled(true)
+            actionBar.elevation = 0f // remove shadow
+        }
+
+        val contactUsButton = findViewById<TextView>(R.id.contact_us_button)
+        contactUsButton.setOnClickListener { createNewZendeskTicket() }
```
Comment (0nko) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt]: You could use Android Kotlin extensions here and avoid inflating the views manually by using `import kotlinx.android.synthetic.main.help_activity.*`. Just note that you'll have to rename the view ids in the layout XML if you want camel-case variable names.

Comment (oguzkocer) [reply WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt]: That sounds like a good idea. Do we already use that pattern elsewhere in the app? I don't want to do something inconsistent as part of Zendesk work.

**Comment by 0nko** [comment]
:shipit:

**Comment by SylvesterWilmott** [comment]
>I just realized that we still have the To continue please enter your email address message for the Contact email in the Help page. I feel like that's probably unnecessary. Should we just show a title like Contact email when it's being changed? Or a different kind of explanation maybe?

@oguzkocer Yeah, let's use `Please enter your email address` here

**Pull request merged**